composer.json

```diff
@@ -24,13 +24,14 @@
         }
     ],
     "require": {
-        "php": "^7.0",
+        "php": "^7.1.3",
```

Comment:
We should be careful here: Symfony 3.4 would e.g. bump to "twig/twig": "^1.43|^2.12", people on php 7.0 would be forced to downgrade to Twig 1 in order to continue receiving Symfony upgrades.

Reply:
I have one client who will be locked on Twig 2.11 now. I don't want to complain becuase nobody really should be on php 7.0 anymore these days.

Anyway, if that change is kept, we should at least document the bump in the ChangeLog.

Reply:
see  #3157